src/Uno.CrossTargeting.props

```diff
@@ -1,6 +1,6 @@
 <Project ToolsVersion="15.0">
-	<Import Project="$(SolutionDir)\crosstargeting_override.props" Condition="exists('$(SolutionDir)\crosstargeting_override.props')" />
-
+	<!-- <Import Project="$(SolutionDir)\crosstargeting_override.props" Condition="exists('$(SolutionDir)\crosstargeting_override.props')" /> -->
+	<Import Project="crosstargeting_override.props" Condition="exists('crosstargeting_override.props')" />
```

Comment:
Why did you need this change ?

Reply:
For some reason having the $(SolutionDir) prefix wasn't working - you might have some more insights on this and want to reverse it.